docker/dev.Dockerfile

```diff
@@ -94,8 +94,9 @@ RUN rsync -v --recursive --chmod=D2775,F444 \
     --exclude '*.go' --exclude 'puntmgr*' --exclude 'cnfreg*' --exclude 'mockcnf*' proto/ /api/
 RUN /usr/local/bin/stonework-init --print-spec > /api/models.spec.yaml
 
-## Install agentctl
-COPY --from=vppagent /bin/agentctl /usr/local/bin/agentctl
+# Install agentctl
+RUN go install go.ligato.io/vpp-agent/v3/cmd/agentctl@master
```

Comment:
We do not want agentctl version at master branch. Just omit the `@master` to install the agentctl at version defined in `go.mod` file. That way we install the same version we are using for the vpp-agent packages. This way we can be more certain to be compatible, compared to the previous behavior which just copied agentctl from the base docker image that is most likely older than the version from `go.mod` file.

Reply:
Done.